### PR TITLE
Fix URL handling in plugin API generation

### PR DESCRIPTION
**Pull request by rwfeather**
Fix for https://github.com/meltano/hub/pull/876#issuecomment-1268791630

### CODE REVIEW

File: src/api/plugins.js [hunk 1]
```diff
@@ -165,23 +165,23 @@ module.exports = function buildJSONApi(gridsome) {
             result.data.data.edges.map(async ({ node: plugin }) => {
               // Clean up plugin object, create full log_url, add docs url
               const logoUrl = plugin.logo_url
-                ? path.join(baseurl, plugin.logo_url)
+                ? new URL(plugin.logo_url, baseurl)
```
Comment (edgarrmondragon) [inline src/api/plugins.js:168]: I think gridsome is pre-processing the logo images so these URLs are not valid. For example: https://deploy-preview-877--meltano-hub.netlify.app/assets/logos/extractors/stackexchange.png.

That seems unrelated to the base URL issues, though.

**Review by edgarrmondragon — APPROVED** [review]

File: src/api/plugins.js (hunk 1 above)
Comment (rwfeather) [reply src/api/plugins.js:168]: Yeah you're right. #878 should resolve the issue
